### PR TITLE
[CA-1479] Move to googleProject for pet SA calls

### PR DESCRIPTION
**Pull request by s-rubenstein**
<!--
Hello, friend!
Remember to mention what sort of testing/verification you performed in the course of building this PR, i.e. checking functionality in the browser locally.
Thanks!
--!>

### CODE REVIEW

**Review by ehigham — APPROVED** [review]
Looks good. Like the parameter reordering.

File: src/components/IGVBrowser.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ const IGVBrowser = _.flow(
           return knownBucketStatus
         } else {
           try {
-            await Ajax(signal).Buckets.getObject(bucket, file, workspace.workspace.namespace, { fields: 'kind' })
+            await Ajax(signal).Buckets.getObject(workspace.workspace.googleProject, bucket, file, { fields: 'kind' })
```
Comment (jacmrob) [inline src/components/IGVBrowser.js:36]: does order matter here? previously it went bucket, file, namespace (project), new one is project, bucket, file

Comment (jacmrob) [reply src/components/IGVBrowser.js:36]: looking at the rest of the code, it seems intentional. just curious why it could be re-ordered

**Review by gpolumbo-broad — APPROVED** [review]
LGTM, but how do we test all of this?

**Review by dmohs — APPROVED** [review]
Very well-contained PR, making it easy to review.

**Comment by s-rubenstein** [comment]
@dmohs do you have any clear advice on testing all of this? I clicked around the UI some, and nothing seemed obviously broken (and it compiled), but especially with some of the data table pieces I wasn't sure how to test it.

**Review by jacmrob — COMMENTED** [review]
I noticed there are still instances where `namespace` sticks around - is that intended?

**Comment by breilly2** [comment]
There were previously some cases where we would use a general pet service account instead of a project-specific service account because, for popular public workspaces, it was too easy to hit the project's service account limit. That may be less of a problem in a PPW world, but I don't imagine it going away completely. Is there a plan to mitigate this?

**Comment by dmohs** [comment]
@s-rubenstein

> @dmohs do you have any clear advice on testing all of this?

The integration tests cover critical use cases. It's not feasible to test every page manually. Spot check where you can (and get help if there is something you'd like to test but can't figure out how to do so). Then, take comfort in the fact that this codebase is built to minimize mean time to repair, so if something does break in this PR or another, we can usually fix it quickly.

**Comment by s-rubenstein** [comment]
@jacmrob namespace is still used as a workspace identifier. So we still want it for things where:
A) We do routing
B) We log mixpanel events
C) We make calls to rawls to fetch data in a workspace.

@breilly2 I am not aware of any mitigation for this, but I don't think it affects this change, since this is only changing ones that are getting the SA token by passing in namespace. 

@dmohs good to know. Yeah, my inclination was to do some minimal smoke testing and then fix in post if things are broken, given the tests all passed.

**Comment by breilly2** [comment]
Ah, my mistake. It looked to me like there were a couple of places where `googleProject` was added where there wasn’t previously `namespace`, but I think I tripped over argument reordering. Sorry about that.

**Pull request merged**